changes/support-apple-m1

```diff
@@ -0,0 +1 @@
+* Add support for Apple M1 processors for development.
```

Comment:
Do you think changes like this that effect only the development experience should make it into the changelog? I'm not opposed, but I want to make sure we are being consistent.

Reply:
Hm... I guess it depends on who we want the audience of the changelog to be. Development changes are important for contributors, but maybe those should fly under the radar changelog-wise.

Reply:
I think they perhaps should be left out of the changelog -- Contributors are usually working off the `main` branch anyway and would likely not be seeing the changelog until well after changes are merged to `main`.